pkg/webui/console/views/gateway-overview/gateway-overview.js

```diff
@@ -77,6 +116,21 @@ export default class GatewayOverview extends React.Component {
       },
     } = this.props
 
+    const global_conf_str = JSON.stringify(this.state.globalConf)
```

Comment:
```suggestion
    const globalConfJson = JSON.stringify(this.state.globalConf)
```

Always use camel case.